### DISCUSSION TITLE
Can headless be disabled on Linux?

### DISCUSSION THREAD

**Discussion by krypterro** [seleniumbase/SeleniumBase#2189] [Q&A] [answered]
This basic code is still headless, how can I adjust it to see the GUI on a Linux desktop?

```
from seleniumbase import Driver
import time

driver = Driver(
    uc=True,
    headless=False
    )
driver.get("https://nowsecure.nl/#relax")
time.sleep(6)
driver.quit()

```

**Comment by mdmintz** [comment, marked as answer]
You're looking for `headed=True`:

https://github.com/seleniumbase/SeleniumBase/blob/ba7f21a3e93e4fd742d29a107a9d84814207c553/seleniumbase/plugins/driver_manager.py#L69

Updated code: (The `driver` also has the `sleep(seconds)` method included)

```python
from seleniumbase import Driver

driver = Driver(uc=True, headed=True)
driver.get("https://nowsecure.nl/#relax")
driver.sleep(6)
driver.quit()
```

**Comment by krypterro** [comment]
Yep, that does the trick, thank you.
